### DISCUSSION TITLE
Login problem

### DISCUSSION THREAD

**Discussion by Chirag-S-Kotian** [vercel/community#6680] [Account Issue] [answered]
I can't login to my vercel account by github or by my gmail help me anyone please

**Comment by amyegan** [comment, marked as answer]
Hey @Chirag-S-Kotian. We can't safely discuss account information in public because of security concerns. If you have not already, please send a message to [registration@vercel.com](mailto:registration@vercel.com) for help with this.

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)

**Reply by Chirag-S-Kotian** [reply]
There is nothing info why I can't login and i already contacted vercel team but I didn't get any proper explanation about my issue
